modules/ROOT/pages/genai-integrations.adoc

```diff
@@ -266,9 +266,46 @@ Supported values:
 
 | model
 | STRING
-| The ID of the model you want to invoke. +
-Supported values: `"text-embedding-ada-002"`
-| "text-embedding-ada-002"
+| The name of the model you want to invoke.
+| `"text-embedding-ada-002"`
+
+| dimensions
+| INTEGER
+| The number of dimensions you want to reduce the vector to. Only supported for certain models (see official documentation).
+| Model-dependent.
+|===
+
+=== Azure OpenAI
+
+* Identifier: `"AzureOpenAI"`
```

Comment:
we call it `Identifier` here (and for the rest) … thinking about this, shouldn't this note that this is the "provider name" to tied to more closely to the interface?

"provider name identifier" perhaps?

Reply:
Yeah honestly I wasn't sure if this was clear enough when I wrote it originally. I tried to clarify by documenting the `provider` parameter as "the case-insensitive identifier of the provider to use". Not a huge fan of "provider name identifier" ("name identifier" seems a bit redundant) but maybe just making it very explicit that it is the corresponding argument for the `provider` parameter?

Reply:
yeah it's a bit much, what do you have in mind for the explicit note?

Reply:
Something like
```suggestion
* Identifier (`provider` argument): `"AzureOpenAI"`
```
perhaps?

Reply:
Hmmnnn, seems a little odd, what not just
```adoc
`provider` argument: `'AzureOpenAI'`
```

Reply:
I'd argue that still calling it "identifier" is useful to match the documentation of the parameter. I imagined "`provider` argument" as just clarification of where it's applicable :)